source/overview/features.rst

```diff
@@ -0,0 +1,282 @@
+============================
+Features Overview
+============================
+
+This page gives an overview of some of the features available in Mattermost Team and Enterprise Editions.
+
+Team Edition Features
+----------------------------------------------
+
+----------------------------------------------
+
+.. raw:: html
+
+    <table class="features__table">
+        <tbody>
+            <tr>
+                <td>
+                    <h4>1-1 and group messaging</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35308655-99c4cb02-0076-11e8-90ae-bda096160995.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Chat in real time with individuals and teams. See all your conversations in one place.</p>
```

Comment:
0/5 Description doesn't match the title of this section since it references chatting with "teams". We could call this section "real time messaging" and then put direct, group and team conversations in the description?

Reply:
Also 0/5 but I thought we wanted to move away from the word "chat" > "communication"

Reply:
From Chen: "Changed 'chat' to 'collaborate' - IMO "messaging" connotes "real-time". I agree we can tighten this up in the future."